tests/integration/models/device.spec.js

```diff
@@ -2559,6 +2559,24 @@ describe('Device Model', function () {
 						expect(result).to.be.empty;
 					});
 
+					it.only(`should expand actor attributes when getting device history by ${testSet.model}`, async function () {
+						const result = await historyModel[testSet.method](
+							this[testSet.model]['id'],
+							{
+								fromDate: subDays(new Date(), 1),
+								toDate: addDays(new Date(), 1),
+								$top: 1,
+								$orderby: 'created_at asc',
```

Comment:
How about moving the $orderby last in the object like we do in the API?

Reply:
Thx!